internal/pkg/deploy/cloudformation/stack/workload.go

```diff
@@ -198,7 +193,7 @@ func (w *wkld) addonsOutputs() (*template.WorkloadNestedStackOpts, error) {
 	stack, err := w.addons.Template()
 	if err != nil {
 		var notFoundErr *addon.ErrAddonsNotFound
-		if !errors.As(err, &notFoundErr) {
+		if !errors.As(err, &notFoundErr){
```

Comment:
```suggestion
		if !errors.As(err, &notFoundErr) {
```
I think this might also get caught by gofmt